mlir/lib/Dialect/MIOpen/Transforms/AlignTiling.cpp

```diff
@@ -317,6 +339,12 @@ static Value reconfigureLAGeneric(PatternRewriter &b,
       Value newInput;
       if (inp == twout) {
         newInput = laIn;
+      } else if (auto expanded = inp.getDefiningOp<memref::ExpandShapeOp>()) {
```

Comment:
You can eliminate this special case by calling traceToThreadwiseCopy to check if this is the threadwisecopy output.